module/.jest/__tests__/utils/arrays.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { Arrays } from '../../../src/utils/arrays';
```

Comment:
Should these utils really be in armstrong, and not a separate lib?

Reply:
I think that's a question for other people, not this PR.